src/LanguageServer.ts

```diff
@@ -740,7 +740,7 @@ export class LanguageServer {
         //clone the diagnostics for each code action, since certain diagnostics can have circular reference properties that kill the language server if serialized
         for (const codeAction of codeActions) {
             if (codeAction.diagnostics) {
-                codeAction.diagnostics = codeAction.diagnostics.map(x => util.toDiagnostic(x, params.textDocument.uri));
+                codeAction.diagnostics = codeAction.diagnostics?.map(x => util.toDiagnostic(x, params.textDocument.uri));
```

Comment:
Behavioral change: If `codeActions.diagnostics` is `undefined` the changed code will silently ignore the error rather than throwing. I am unsure of whether this can happen in practice.